### ISSUE TITLE
Remove item_names.lua

### ISSUE THREAD

**Issue by Wuzzy2** [minetest-mods/unified_inventory#101]
I suggest to remove `item_names.lua`. This is the functionality which modifies the HUD (!) by displaying the wielded item name display above the statbars.

I think this feature is out of place for an inventory mod. IMO this function is better provided in a separate mod.

In fact, I have already done this by extracting the mod code and adding my own features:
https://forum.minetest.net/viewtopic.php?f=9&t=18011

**Comment by VanessaE** [comment]
This project has been moved to Gitlab.  Please copy/paste/re-open your issue there.  https://gitlab.com/VanessaE/unified_inventory/issues

**Comment by Wuzzy2** [comment]
The project on GitLab is dead. Re-open?

**Comment by SmallJoker** [comment]
Yes sure, although I do not support your request and think the HUD fits perfectly into this inventory management mod.

**Comment by Wuzzy2** [comment]
The problem is, it's impossible to get rid of this once UI is installed.

**Comment by SmallJoker** [comment]
I could say the same thing once show_wielded_item is installed. It breaks UI and cannot be disabled by UI.

This collision seems to be unavoidable, just like forks of the same mod.

**Comment by rusty-snake** [comment]
unified_invertory is the best minetest mod for the inventory I know. There a only a few things which bother me, the icons for bags and others are not pixelated enough IMO, the buttons/background has its own style. And the text which shows the wielded item, which is IMO useless.

I would really welcome a setting to disable this.

**Comment by VanessaE** [comment]
The icons are intentionally high resolution.  The thinking at the time was that they should look good even on so-called "retina" displays, and that there was no real, legit reason for them to look pixelated.  This isn't 1985.

RealBadAngel, @ShadowNinja, and I put a lot of effort into making them look nice :smile:

**Comment by rusty-snake** [comment]
> and that there was no real, legit reason for them to look pixelated. This isn't 1985.

They catch the eye, because all other icons in minetest are pixelated.

**Comment by VanessaE** [comment]
"all other icons" are pixelated because lack of artistic skills in the mods supplying them :smile:

**Comment by SmallJoker** [comment]
Closing in favour of #135.
